api_tests/dry/rfc003_swap_reject.ts

```diff
@@ -268,7 +277,7 @@ describe("SWAP request REJECTED", () => {
         }
```

Comment:
Not your code, but above, line 266 
`let swap_2 = await chai.request(bob.comit_node_url()).get(swap_2_href);`
is unused.

Reply:
Yep, already removed :)